about.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Digital Master Plan 2023</title>
+  <link rel="stylesheet" href="css/general.css">
+  <link rel="stylesheet" href="css/about.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+  </style>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+  <my-menu></my-menu>
+  <section class="section d-flex-column d-ds-flex-column justify-content-center align-items-center container" id="about">
+    <div>
+      <h1 class="title">
+        digital master plan global summit 2023
+      </h1>
+  
+      <p class="para">
+        The global summit brings together the community of experts, academics, and activist who comprise the Creative Commons affliate network in a different country every two, This year, we hope to expand our invitation list including organizations and individuals.
+      </p>
+  
+      <div>
+        <p class="txt-details">
+          please contact us per any Email for any Further questions
+          about Digital Master Plan<br>
+          <span class="address-info">digital@info.ac.ke</span>
+        </p>
+      </div>
+    </div>
+  </section>
```

Comment:
Good job with the about page so far so far 👍👍
- [ ] I noticed that there is no background image added to this section. Please consider adding a background image to the section so that we have a design close to the design guidelines according to the requirement 🔧🤓 Please make sure to follow the design guidelines
- [ ] Also, I noticed a difference in the `font-size`, `font-weight`, `padding/margin`, and `test-alignment` between your design and the design guidelines. Please consider increasing the font size and adjust the font weight and padding/margin. Also, please consider fixing the design/style of `<h1>`. ⚒🔍 For more clarity please visit the above comments 👆
- [ ] Please consider using the same amount of text under the heading `<h1>` as given in the design guidelines

### _Desktop version 🔽_
Your design | Design guidelines |
--- | --- | 
![image](https://user-images.githubusercontent.com/106119105/217341409-d0db8fb0-44db-4ada-b6df-ecff89fc4dd0.png) | ![image](https://user-images.githubusercontent.com/106119105/217341072-bf00977a-bbc5-484c-bf7a-c4f22b4da435.png) | 

## **_Mobile version 🔽_**
Your design | Design guidelines |
--- | --- | 
![image](https://user-images.githubusercontent.com/106119105/217341971-f95e4f44-c815-4f90-a25f-6057b1caf93a.png) | ![image](https://user-images.githubusercontent.com/106119105/217342068-e2276d92-d238-418a-97d6-f1668d8e353b.png) |